content/02.main-text.md

```diff
@@ -110,7 +110,9 @@ Manubot includes an additional layer of citation processing, currently unique to
 All citations point to a standard identifier, for which Manubot automatically retrieves bibliographic metadata.
 Table @tbl:citations presents the supported identifiers and example citations before and after Manubot processing.
 Authors can optionally define citation tags to provide short readable alternatives to the citation identifiers.
-Metadata is exported to [Citation Style Language (CSL) JSON Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#items), an open standard that is widely supported by reference managers [@doi:10.1007/978-3-319-00026-8_8; @doi:10.1080/02763869.2012.641841].
+Metadata is exported to [Citation Style Language (CSL) JSON Data Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#items), an open standard that is widely supported by reference managers [@doi:10.1007/978-3-319-00026-8_8; @doi:10.1080/02763869.2012.641841].
+However, sometimes external resources provide Manubot with invalid CSL Data, which can crash downstream citation processors, such as [pandoc-citeproc](http://hackage.haskell.org/package/pandoc-citeproc).
+Therefore, Manubot removes invalid fields according to the [CSL Data specification](https://github.com/citation-style-language/schema).
```

Comment:
```suggestion
Therefore, Manubot removes invalid fields according to the [CSL data specification](https://github.com/citation-style-language/schema).
```

Reply:
`CSL Data` appears three times.  If we change it here, we should change it everywhere for consistency.  Lower case `data` is okay with me if that isn't a formal proper name.